src/aqdata/tests/lsgc_test.cc

```diff
@@ -0,0 +1,77 @@
+#include "gtest/gtest.h"
```

Comment:
`#include` should be subject to the following ordering: project headers->STL headers->third-party headers with one empty line separating each block. gtest is third-party so it should be put to the bottom